Tests/SwiftDocCTests/Rendering/HeadingAnchorTests.swift

```diff
@@ -0,0 +1,59 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021-2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import XCTest
+@testable import SwiftDocC
+@_spi(FileManagerProtocol) import SwiftDocCTestUtilities
```

Comment:
nit: This SPI import isn't needed anymore after we moved the type to use `package` access.
```suggestion
import SwiftDocCTestUtilities
```

Reply:
Fixed